config/initializers/assets.rb

```diff
@@ -11,4 +11,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w( admin.js admin.css )
```

Comment:
is there some risk to dishing up this jabbascript with the non admin pages?

Reply:
Risk of dishing it out, yeah, risk of it being dangerous: nah.